marketplace_builder/views/pages/how-to/amp/feeding-data-amp-pages-graphql.liquid

```diff
@@ -0,0 +1,178 @@
+---
+converter: markdown
+metadata:
+  title: Feeding Data into AMP Pages with GraphQL
+  description: This guide will help you feed data into your AMP pages using GraphQL. 
+slug: how-to/amp/feeding-data-amp-pages-graphql
+searchable: true
+---
+
+This guide will help you feed data into your AMP pages using GraphQL. The guide explains the process through a real-life example of displaying products of an online store.  
+
+## Requirements
+This is an advanced tutorial. To follow it, you should be familiar with basic Platform OS concepts, technologies, and the topics in the Get Started section, especially pages. You should also know what AMP pages are and how to create them.  
+
+* [How Platform OS Works](/how-platformos-works)
+* [Get Started: Pages](/get-started/pages/pages)
+* [Using a Static AMP Template](/how-to/amp/using-static-amp-template)
+* [Converting AMP HTML Pages to Layouts and Pages](/how-to/amp/converting-amp-html-layouts-pages)
+ 
+## Steps
+Feeding data into AMP pages with GraphQL, in this case, displaying products on a page is a four-step process:
+
+1. Display list of products 
+2. Create JSON endpoint
+3. Fetch listing data
+4. Display product details
+
+### Step 1: Display list of products
+
+Create the main page, HTML extended with AMP tags and objects `marketplace_builder/views/pages/product-list.liquid`:
+
+<pre><code class="language-liquid">
+---
+slug: product-list
+format: html
+layout_name: application
+---
+<!-- AMP dependencies -->
+<script custom-element="amp-list" src="https://cdn.ampproject.org/v0/amp-list-0.1.js" async=""></script>
```

Comment:
Script tags need to be in content_for and this content_for needs to be yielded in the layout for this to work.

Something like this:
https://github.com/mdyd-dev/amp-demo-pages/blob/master/travel/marketplace_builder/liquid_views/layouts/application.liquid#L17 +
https://github.com/mdyd-dev/amp-demo-pages/blob/master/travel/marketplace_builder/pages/travel.liquid#L7-L11

Reply:
@pavelloz  You are right, I was just trying to keep it short. Changed now. (Lucian)